pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pytrends"
+version = "4.7.4"
+description = "Pseudo API for Google Trends"
+urls = {homepage = "https://github.com/dreyco676/pytrends"}
+requires-python = ">=3.7"
+authors = [
+    {name = "John Hogue"},
+    {name = "Burton DeWilde"},
+    {email = "dreyco676@gmail.com"}
+]
+license = {text = "Apache 2.0"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3.3",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
```

Comment:
I'm okay with removing these.

Reply:
Oops, an oversight, I'll update them.

Reply:
I fixed more things in the `pyproject.toml`:
* updated the license classifier to match with the `LICENSE` file.
* updated the version to match the latest release at [pypi](https://pypi.org/project/pytrends/) (4.8.0).